dali/pipeline/data/tensor_vector.cc

```diff
@@ -62,11 +63,112 @@ TensorVector<Backend>::TensorVector(TensorVector<Backend> &&other) noexcept {
   other.tensors_.clear();
 }
 
-
 template <typename Backend>
-size_t TensorVector<Backend>::nbytes() const noexcept {
+void TensorVector<Backend>::UnsafeSetSample(int dst, const TensorVector<Backend> &owner, int src) {
```

Comment:
```suggestion
void TensorVector<Backend>::UnsafeSetSample(int sample_idx, const TensorVector<Backend> &owner, int src_sample_idx) {
```

Reply:
I prefer to keep the consistent for the documentation purposes.